backend/tests/test_exceptions.py

```diff
@@ -153,3 +153,24 @@ def test_update_db_user_no_results(test_client, db, setup_admin_session):
 def test_delete_db_user_no_results(test_client, setup_admin_session):
```

Comment:
Should we also add some asserts to `test_non_admin_access_admin_endpoint` for permissions on new endpoints?

Reply:
Resolved in latest commit.